### ISSUE TITLE
Change the loss function

### ISSUE THREAD

**Issue by juanabascal** [juanabascal/tf-flower-classificator#5]
Change the loss function to sparse softmax cross entropy with logits.

**Comment by dgdelahera** [comment]
Done and pushed to master
